parmed/openmm/parameters.py

```diff
@@ -101,10 +107,20 @@ def from_parameterset(cls, params, copy=False):
         new_params.parametersets = params.parametersets
         new_params._combining_rule = params.combining_rule
         new_params.residues = params.residues
+        # Assign Lennard-Jones atom classes
+        ljtypes = dict()
+        counter = 0
+        for name, atom_type in iteritems(new_params.atom_types):
+            key = (atom_type.rmin, atom_type.epsilon, atom_type.rmin_14,
+                    atom_type.epsilon_14)
+            if key not in ljtypes:
+                ljtypes[key] = counter
```

Comment:
What if we instead use the text name of the atom type?  For example, the [relevant section of the `parm96.dat` file](https://github.com/choderalab/ambermini/blob/master/share/amber/dat/leap/parm/parm96.dat#L450-L451) specifies that a number of types map to `N` or `C`:
```
N   NA  N2  N*  NC  NB  NP  NO
C   C*  CA  CB  CC  CN  CM  CK  CQ  CW  CV  CR  CA  CX  CY  CD
```
while other LJ types are specified for their standard Amber type names.  It might be good to keep this mapping for `ljtypes`, rather than introducing a confusing new atom type numbering scheme that is used for OpenMM `class` assignment below.